ec2/sc-ec2-linux-docker.yaml

```diff
@@ -343,3 +343,6 @@ Outputs:
   EC2ConsoleURI:
     Description: 'Check your instance status with this link to the AWS Console'
     Value: !Sub "https://console.aws.amazon.com/ec2/v2/home?region=${AWS::Region}#Instances:search=${EC2Instance}"
+  Documentation:
+    Description: 'Service Catalog Documentation'
+    Value: "https://sagebionetworks.jira.com/wiki/spaces/SC/pages/938836322/Service+Catalog+Provisioning"
```

Comment:
The Confluence link is private, only accessible to Sage employees. Please use the Scroll Viewport URL which is public (and looks nicer):
https://help.sc.sageit.org/sc/Service-Catalog-Provisioning.938836322.html

Reply:
thanks, atlassian is so confusing. anyways, it's fixed now.